lib/awful/widget/tasklist.lua

```diff
@@ -461,15 +511,16 @@ function tasklist.new(args, filter, buttons, style, update_function, base_widget
         w:set_spacing(args.style and args.style.spacing or beautiful.tasklist_spacing)
     end
 
+    local queued_update = false
```

Comment:
Oh, I didn't even see this. This is in the wrong commit. The variable `queueued_update` should be moved in the commit that adds `_do_tasklist_update_now`.